generator/template/process.go

```diff
@@ -169,9 +172,38 @@ func processTableSQLBuilder(fileTypes, dirPath string,
 
 		err = utils.SaveGoFile(tableSQLBuilderPath, tableSQLBuilder.FileName, text)
 		throw.OnError(err)
+
+		generatedBuilders = append(generatedBuilders, tableSQLBuilder)
+	}
+
+	if len(generatedBuilders) > 0 {
+		generateUseSchemaFunc(dirPath, fileTypes, schemaMetaData, generatedBuilders)
 	}
 }
 
+func generateUseSchemaFunc(dirPath, fileTypes string, schemaMetadata metadata.Schema, builders []TableSQLBuilder) {
+
+	basePath := path.Join(dirPath, builders[0].Path)
+
+	schemaIdentifier := utils.ToGoIdentifier(schemaMetadata.Name)
```

Comment:
My [comment](https://github.com/go-jet/jet/pull/194#discussion_r1038840362) was mainly about using the current schema identifier in the function name. Because we are replacing 'dvds' as schema name not setting. So I'm thinking it's better to go without a schema identifier in the function name completely.

Reply:
@go-jet `tests/mysql/update_test.go` has a dot import for both `.gentestdata/mysql/dvds/table` and `.gentestdata/mysql/test_sample/table`. Simply generating `UseSchema` would cause a "Function `UseSchema` redeclared in this package" error. This was my initial reasoning behind using the schema identifier in the generated method.

Reply:
Aha, I see. We can extract test related to `.gentestdata/mysql/dvds/table` into separate `update_dvds_test.go` file. There is just one test there - `TestUpdateWithJoin`.

Reply:
Ah this makes sense. Perfect!

Reply:
@go-jet I'm wondering why CircleCI tests are failing. These tests work just fine when run with `go run ./...`.

Reply:
It seems that tableMetaData slice is unordered, so on different machines slightly different file is constructed. Adding `ORDER BY table_name` [here](https://github.com/go-jet/jet/blob/c5e3d1ee7505a16b0fcb4fcf09d89e64ca5d0adf/generator/postgres/query_set.go#L18) and [here](https://github.com/go-jet/jet/blob/c5e3d1ee7505a16b0fcb4fcf09d89e64ca5d0adf/generator/mysql/query_set.go#L19), should probably fix the tests.

Reply:
Great. It looks good. :+1: